edit_course/operations/configure.py

```diff
@@ -211,15 +211,15 @@ def configure_content(instance, url):
         url = url.strip()
         response = requests.get(url)
     except Exception as e:
-        return [_("Request failed: {}").format(str(e))]
+        return [_("Request failed: {}.").format(str(e))]
```

Comment:
`_("Request for a course configuration failed with an error '{error!s]', and thus configuration of a course was aborted.").format(error=e)`